back/app/models/custom_field.rb

```diff
@@ -146,13 +154,28 @@ def accept(visitor)
       visitor.visit_linear_scale self
     when 'page'
       visitor.visit_page self
+    when 'section'
+      visitor.visit_section self
     when 'file_upload'
       visitor.visit_file_upload self
     else
       raise "Unsupported input type: #{input_type}"
     end
   end
 
+  # Special behaviour for ideation section 1
+  def title_multiloc
+    if key == 'ideation_section_1'
```

Comment:
Ah, wasn't sure why we had two fields when they all seem to be the same, but would appear that's not the case. Have changed

Reply:
Yes, I've been confused about the code and key for a long time :p The code is used to identify built-in fields, the key is derived from the title (I think in some cases it can even be chosen by the admin) and is used as key in the `custom_field_values`.